### PR TITLE
Update README: feature list, document hyperlinks and getting started guide

### LINKED ISSUE #188: Update README from astropy package template
[README.rst](https://github.com/skypyproject/skypy/blob/master/README.rst) is missing some key information:
- Summary of SkyPy features
- ~"Contributing" should state that we are currently not accepting external contributions~
- Links to [LICENSE.rst](https://github.com/skypyproject/skypy/blob/master/LICENSE.rst), [CODE_OF_CONDUCT.md](https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md), [CONTRIBUTING.md](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md), [CITATION](https://github.com/skypyproject/skypy/blob/master/CITATION)
- Requirements, how to install, running tests, importing the package, running examples
- Any other ideas?

### PR DESCRIPTION
**Pull request by Lucia-Fonseca**
## Description
This pull request update the [README.rst](https://github.com/skypyproject/skypy/blob/master/README.rst) with some key information:
- Summary of SkyPy features
- Links to ~[LICENSE.rst](https://github.com/skypyproject/skypy/blob/master/LICENSE.rst)~, [CODE_OF_CONDUCT.md](https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md), [CONTRIBUTING.md](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md), [CITATION](https://github.com/skypyproject/skypy/blob/master/CITATION)
- Requirements, how to install, running tests, importing the package, running examples

This pull request resolves #188 .
## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [ ] Write documentation strings
- [ ] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -8,6 +8,48 @@ This package contains methods for modelling the Universe, galaxies and the Milky
...
+The full list of features can be found on the
+[SkyPy Documentation](https://skypy.readthedocs.io/en/latest/?badge=latest) main page
+* Clusters
+* Galaxies
+* Gravitational waves
+* Dark Matter Halos
+* Power Spectrum
+* Supernovae
+* Utiles
+* Pipeline
```
Comment (rrjbca) [inline README.rst:23]:
```suggestion
The full list of features can be found on the
[SkyPy Documentation](https://skypy.readthedocs.io/en/latest/?badge=latest) and includes:

* Clusters
* Galaxies
* Gravitational Waves
* Dark Matter Halos
* Power Spectrum
* Supernovae
* Utilities
* Pipeline
```

File: README.rst [hunk 2]
```diff
@@ -8,6 +8,48 @@ This package contains methods for modelling the Universe, galaxies and the Milky
...
+
+
+Important links
+---------------
+
+* [License](https://github.com/skypyproject/skypy/blob/master/LICENSE.rst)
+* [Code of Conduct](https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md)
+* [Contributing guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
+* [Citation guidelines](https://github.com/skypyproject/skypy/blob/master/CITATION)
```
Comment (rrjbca) [inline README.rst:33]:
```suggestion
* `License <https://github.com/skypyproject/skypy/blob/master/LICENSE.rst>`_
* `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md>`_
* `Contributor Guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)>`_
* `Citation Guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION>`_
```

**Comment by Lucia-Fonseca** [comment]
This should be ready to review

File: README.rst [hunk 3]
```diff
@@ -8,6 +8,69 @@ This package contains methods for modelling the Universe, galaxies and the Milky
...
+* Pipeline
+
+
+Important links
+---------------
+
+* `License <https://github.com/skypyproject/skypy/blob/master/LICENSE.rst)>`_
+* `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md)>`_
+* `Contributing guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)>`_
+* `Citation guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION)>`_
```
Comment (rrjbca) [inline README.rst:33]:
```suggestion
* `Code of Conduct <https://github.com/skypyproject/skypy/blob/master/CODE_OF_CONDUCT.md>`_
* `Contributor Guidelines <https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md>`_
* `Citation Guidelines <https://github.com/skypyproject/skypy/blob/master/CITATION>`_
```

Comment (Lucia-Fonseca) [reply README.rst]: Good spot! Thanks. This is fixed now.

File: README.rst [hunk 4]
```diff
@@ -8,6 +8,69 @@ This package contains methods for modelling the Universe, galaxies and the Milky
...
+  Make sure you have current version of ``astropy``, ``networkx``,
+  ``numpy``, ``scipy`` and ``pyyaml``.
+
+* Installation with pip:
+
+  ``pip install skypy``
+
+* Installation with conda:
+
+  (coming soon)
```
Comment (rrjbca) [inline README.rst:50]:
```suggestion
  ``conda install -c conda-forge skypy``
```

File: README.rst [hunk 5]
```diff
@@ -8,6 +8,69 @@ This package contains methods for modelling the Universe, galaxies and the Milky
...
+
+* Running tests:
+
+  ``python setup.py test``
```
Comment (rrjbca) [inline README.rst:54]:
```suggestion
  ``pytest --pyargs skypy``
```

**Comment by Lucia-Fonseca** [comment]
Done.

File: README.rst [hunk 6]
```diff
@@ -2,27 +2,91 @@
...
+
+SkyPy also has a driver script that can run pipelines. The repository contains
+examples_ of config files in yaml format to download and run. For example:
+
+.. code:: bash
+
+    skypy –config examples/herbel_galaxies.yaml –format fits
+
+will generate tables of red and blue galaxies following the model described in
+`Herbel et al. 2017`_ and write them in fits format.
```
Comment (ntessore) [inline README.rst:67]:
```suggestion
`Herbel et al. 2017`_ and write them in FITS format.
```

**Review by ntessore — APPROVED** [review]
Potential fix for typo, otherwise fine.

**Review by Lucia-Fonseca — COMMENTED** [review]
Looks very good!

**Review by rrjbca — APPROVED** [review]

**Pull request merged**